src/goto-programs/goto_program_template.h

```diff
@@ -503,7 +503,7 @@ class goto_program_templatet
     const namespacet &ns,
     const irep_idt &identifier,
     std::ostream &out,
-    typename instructionst::const_iterator it) const=0;
+    const typename instructionst::value_type &it) const = 0;
```

Comment:
nit: I don't think we should have spaces around `=`

Reply:
I had let that one go in light of https://github.com/diffblue/cbmc/pull/1324.

Reply:
Yeah, I was not aware of the discussion about the spaces around operators before reviewing this PR.